CondTools/SiPixel/plugins/SiPixelFEDChannelContainerFromQualityConverter.cc

```diff
@@ -135,7 +135,7 @@ SiPixelFEDChannelContainerFromQualityConverter::createFromSiPixelQuality(const S
     std::vector<sipixelobjects::CablingPathToDetUnit> path = theFedCabling.pathToDetUnit(mod.DetID);
     auto cabling_ = theFedCabling.cablingTree();
     unsigned int nrocs_inLink(0);
-    if (path.size() != 0) {
+    if (!path.empty()) {
```

Comment:
This check is useless, if independently of it you later divide by `nrocs_inLink` anyhow.
You can remove it, or (maybe better) throw some exception inside